src/containers/MapExplorer-SelectorPanel.jsx

```diff
@@ -69,7 +69,7 @@ export default class SelectorPanel extends React.Component {
     return (
       <article className="selector-panel">
         <section className={(this.props.selectorData.mode !== SelectorData.GUIDED_MODE) ? 'input-subpanel not-selected' : 'input-subpanel' } ref="subPanel_guided">
-          <h1 onClick={this.changeToGuided}>Standard Mode</h1>
+          <h1 className="hidden" onClick={this.changeToGuided}>Standard Mode</h1>
```

Comment:
Headings can't be focussed with the tab key, so this can only be used with a mouse or touch. Better to use a button for clickable controls like this.